timetasker/main.py

```diff
@@ -117,12 +121,33 @@ def toggle_timer(self):
             self.start()
 
 
+class HelpScreen(ModalScreen):
```

Comment:
Nice one, 
textual also has a command palette automatically available I thought it would be pretty good to integrate with that and just add our commands there https://textual.textualize.io/guide/command_palette/#command-palette

should be able to add by making this a `Provider` instead

Reply:
Just updated to integrate available commands into the palette and generate them from the apps bindings so no duplication and keys added to help for free